### DISCUSSION TITLE
Migrate

### DISCUSSION THREAD

**Discussion by spijker** [apache/hertzbeat#1848] [General]
Is it possible to migrate the data  from H2 to MySQL ? 
Of use an export / import model ?

Great App !

**Comment by tomsun28** [comment]
hi you can export the h2 data sql and import it in mysql, or you can use hertzbeat monitors export import

**Reply by spijker** [reply]
Oh great indeed, just an H2 ;)  , used dbeaver , worked like a charm. Thx
